src/Tools/ExternalAccess/OmniSharp/DocumentationComments/OmniSharpDocumentationCommentsSnippetService.cs

```diff
@@ -13,7 +14,7 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.DocumentationComments
 {
     internal static class OmniSharpDocumentationCommentsSnippetService
```

Comment:
@333fred Can you take a look at the breaking changes for OmniSharp here?

Reply:
Should be fine, these are already used in an async method. @JoeRobich and @filipw, fyi for the next time we upgrade Roslyn we'll need to make a small change here (and yay, the EA assembly is working as expected!).

Reply:
nice, thanks!